cypress/e2e/tests/pages/explorer/dashboard/cluster-dashboard.spec.ts

```diff
@@ -55,9 +55,8 @@ describe('Cluster Dashboard', { testIsolation: 'off', tags: ['@explorer', '@admi
     BurgerMenuPo.checkIfClusterMenuLinkIsHighlighted('local');
   });
 
-  it.skip('[Vue3 Skip]: has the correct title', () => {
-    clusterDashboard.goTo('local');
-    clusterDashboard.waitForPage(undefined, 'cluster-events');
+  it('has the correct title', () => {
+    ClusterDashboardPagePo.navTo();
```

Comment:
The old way to navigate to the local cluster wasn't working anymore. Switched to the way the other tests were navigating to the same page.